mingw-w64-clang/PKGBUILD

```
@@ -220,6 +220,18 @@ build() {
       ;;
   esac
 
+  # List of targets to build in CMake style
+  # For more info see: https://github.com/msys2/MINGW-packages/discussions/9765
+  local _targets="AArch64;AMDGPU;ARM;AVR;BPF;MSP430;NVPTX;RISCV;WebAssembly;X86" 
+
+  # Enable more targets for some system
+  # For more info see: https://github.com/msys2/MINGW-packages/pull/10258
+  for _msystem in MINGW32 MINGW64 UCRT64; do
+    if [[ $MSYSTEM == $_msystem ]]; then
+      _targets="$_targets;Mips"
+    fi
+  done
```

Just stylistically I would use a case statement rather than an if in a loop:
```suggestion
  case "${MSYSTEM}" in
    MINGW*|UCRT*)
      _targets+=";Mips"
      ;;
  esac
```

---

Thanks, my bash isn't very good yet.